CRITICAL - 2024-04-24 05:33:50 --> ErrorException: fopen(path/to/mock.csv): Failed to open stream: No such file or directory
[Method: GET, Route: task1/bulkInsert]
in APPPATH\Controllers\Task1.php on line 13.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'fopen(path/to/mock.csv): Failed to open stream: No such file or directory', 'C:\\xampp\\htdocs\\crud\\app\\Controllers\\Task1.php', 13)
 2 APPPATH\Controllers\Task1.php(13): fopen('path/to/mock.csv', 'r')
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task1->bulkInsert()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task1))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 05:44:12 --> Error: Class "App\Controllers\DuplicateCountModel" not found
[Method: POST, Route: task1/uploadFile]
in APPPATH\Controllers\Task1.php on line 38.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task1->uploadFile()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task1))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-24 05:45:54 --> mysqli_sql_exception: Table 'crud.duplicate_count' doesn't exist in C:\xampp\htdocs\crud\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\crud\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\crud\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\crud\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\crud\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\crud\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\crud\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\crud\app\Controllers\Task1.php(40): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\crud\system\CodeIgniter.php(933): App\Controllers\Task1->uploadFile()
#8 C:\xampp\htdocs\crud\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task1))
#9 C:\xampp\htdocs\crud\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\crud\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\crud\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\crud\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-04-24 05:45:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'crud.duplicate_count' doesn't exist
[Method: POST, Route: task1/uploadFile]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `duplicate_count`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\Task1.php(40): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task1->uploadFile()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task1))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 05:45:54 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'crud.duplicate_count' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `duplicate_count`
WHERE `email` = \'email\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `duplicate_count`
WHERE `email` = \'email\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `duplicate_count`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 6 APPPATH\Controllers\Task1.php(40): CodeIgniter\BaseModel->first()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task1->uploadFile()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task1))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 05:45:54 --> [Caused by] mysqli_sql_exception: Table 'crud.duplicate_count' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *
FROM `duplicate_count`
WHERE `email` = \'email\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `duplicate_count`
WHERE `email` = \'email\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `duplicate_count`
WHERE `email` = \'email\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `duplicate_count`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\Task1.php(40): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task1->uploadFile()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task1))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:31:42 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "task2_view.php"
[Method: GET, Route: Task2]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('task2_view.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('task2_view', [], true)
 3 APPPATH\Controllers\Task2.php(19): view('task2_view')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->index()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:32:03 --> DivisionByZeroError: Division by zero
[Method: GET, Route: task2/getDuplicatePercentage]
in APPPATH\Controllers\Task2.php on line 79.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getDuplicatePercentage()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:32:11 --> ArgumentCountError: Too few arguments to function Firebase\JWT\JWT::encode(), 2 passed in C:\xampp\htdocs\crud\app\Controllers\Task2.php on line 28 and at least 3 expected
[Method: POST, Route: task2/generateToken]
in FCPATH\vendor\firebase\php-jwt\src\JWT.php on line 199.
 1 APPPATH\Controllers\Task2.php(28): Firebase\JWT\JWT::encode([...], 'qwertyuiop')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->generateToken()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-24 06:32:34 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\crud\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\crud\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\crud\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\crud\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\crud\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\crud\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\crud\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\crud\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\crud\app\Controllers\Task1.php(55): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\crud\system\CodeIgniter.php(933): App\Controllers\Task1->uploadFile()
#9 C:\xampp\htdocs\crud\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task1))
#10 C:\xampp\htdocs\crud\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\crud\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\crud\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\crud\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-04-24 06:32:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '0' for key 'PRIMARY'
[Method: POST, Route: task1/uploadFile]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`) VALUES (:email:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Task1.php(55): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task1->uploadFile()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task1))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:32:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`email`) VALUES (\'jane@example.com\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`email`) VALUES (\'jane@example.com\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`) VALUES (:email:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Task1.php(55): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task1->uploadFile()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task1))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:32:34 --> [Caused by] mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `users` (`email`) VALUES (\'jane@example.com\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`email`) VALUES (\'jane@example.com\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`email`) VALUES (\'jane@example.com\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`) VALUES (:email:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Task1.php(55): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task1->uploadFile()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task1))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:36:14 --> TypeError: CodeIgniter\Database\BaseBuilder::limit(): Argument #1 ($value) must be of type ?int, string given, called in C:\xampp\htdocs\crud\system\Model.php on line 919
[Method: GET, Route: task2/getUserDetails]
in SYSTEMPATH\Database\BaseBuilder.php on line 1494.
 1 SYSTEMPATH\Model.php(919): CodeIgniter\Database\BaseBuilder->limit('10', 0)
 2 APPPATH\Controllers\Task2.php(53): CodeIgniter\Model->__call('limit', [...])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:36:37 --> TypeError: CodeIgniter\Database\BaseBuilder::limit(): Argument #1 ($value) must be of type ?int, string given, called in C:\xampp\htdocs\crud\system\Model.php on line 919
[Method: GET, Route: task2/getUserDetails]
in SYSTEMPATH\Database\BaseBuilder.php on line 1494.
 1 SYSTEMPATH\Model.php(919): CodeIgniter\Database\BaseBuilder->limit('10', 0)
 2 APPPATH\Controllers\Task2.php(53): CodeIgniter\Model->__call('limit', [...])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:36:41 --> ArgumentCountError: Too few arguments to function Firebase\JWT\JWT::encode(), 2 passed in C:\xampp\htdocs\crud\app\Controllers\Task2.php on line 28 and at least 3 expected
[Method: POST, Route: task2/generateToken]
in FCPATH\vendor\firebase\php-jwt\src\JWT.php on line 199.
 1 APPPATH\Controllers\Task2.php(28): Firebase\JWT\JWT::encode([...], 'qwertyuiop')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->generateToken()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:39:56 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "task2_view.php"
[Method: GET, Route: Task2]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('task2_view.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('task2_view', [], true)
 3 APPPATH\Controllers\Task2.php(19): view('task2_view')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->index()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:40:08 --> ArgumentCountError: Too few arguments to function Firebase\JWT\JWT::encode(), 2 passed in C:\xampp\htdocs\crud\app\Controllers\Task2.php on line 35 and at least 3 expected
[Method: POST, Route: task2/generateToken]
in FCPATH\vendor\firebase\php-jwt\src\JWT.php on line 199.
 1 APPPATH\Controllers\Task2.php(35): Firebase\JWT\JWT::encode([...], 'your_secret_key')
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->generateToken()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-04-24 06:47:20 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\crud\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\crud\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\crud\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\crud\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\crud\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\crud\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\crud\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\crud\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\crud\app\Controllers\Task1.php(56): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\crud\system\CodeIgniter.php(933): App\Controllers\Task1->uploadFile()
#9 C:\xampp\htdocs\crud\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task1))
#10 C:\xampp\htdocs\crud\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\crud\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\crud\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\crud\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-04-24 06:47:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '0' for key 'PRIMARY'
[Method: POST, Route: task1/uploadFile]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`) VALUES (:email:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Task1.php(56): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task1->uploadFile()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task1))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:47:20 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`email`) VALUES (\'Zea\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`email`) VALUES (\'Zea\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`) VALUES (:email:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Task1.php(56): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task1->uploadFile()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task1))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:47:20 --> [Caused by] mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `users` (`email`) VALUES (\'Zea\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`email`) VALUES (\'Zea\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`email`) VALUES (\'Zea\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`) VALUES (:email:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Task1.php(56): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task1->uploadFile()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task1))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:58:01 --> ParseError: syntax error, unexpected token "public", expecting ";" or "{"
[Method: GET, Route: task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 46.
 1 SYSTEMPATH\Router\Router.php(562): CodeIgniter\Router\AutoRouter->getRoute('task2/getUserDetails', 'GET')
 2 SYSTEMPATH\Router\Router.php(227): CodeIgniter\Router\Router->autoRoute('task2/getUserDetails')
 3 SYSTEMPATH\CodeIgniter.php(834): CodeIgniter\Router\Router->handle('task2/getUserDetails')
 4 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:58:16 --> ParseError: syntax error, unexpected token "public", expecting ";" or "{"
[Method: GET, Route: task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 46.
 1 SYSTEMPATH\Router\Router.php(562): CodeIgniter\Router\AutoRouter->getRoute('task2/getUserDetails', 'GET')
 2 SYSTEMPATH\Router\Router.php(227): CodeIgniter\Router\Router->autoRoute('task2/getUserDetails')
 3 SYSTEMPATH\CodeIgniter.php(834): CodeIgniter\Router\Router->handle('task2/getUserDetails')
 4 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:58:19 --> ParseError: syntax error, unexpected token "public", expecting ";" or "{"
[Method: GET, Route: task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 46.
 1 SYSTEMPATH\Router\Router.php(562): CodeIgniter\Router\AutoRouter->getRoute('task2/getUserDetails', 'GET')
 2 SYSTEMPATH\Router\Router.php(227): CodeIgniter\Router\Router->autoRoute('task2/getUserDetails')
 3 SYSTEMPATH\CodeIgniter.php(834): CodeIgniter\Router\Router->handle('task2/getUserDetails')
 4 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:58:36 --> ParseError: syntax error, unexpected token "public", expecting ";" or "{"
[Method: GET, Route: task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 46.
 1 SYSTEMPATH\Router\Router.php(562): CodeIgniter\Router\AutoRouter->getRoute('task2/getUserDetails', 'GET')
 2 SYSTEMPATH\Router\Router.php(227): CodeIgniter\Router\Router->autoRoute('task2/getUserDetails')
 3 SYSTEMPATH\CodeIgniter.php(834): CodeIgniter\Router\Router->handle('task2/getUserDetails')
 4 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 06:58:54 --> ParseError: syntax error, unexpected token "public", expecting ";" or "{"
[Method: GET, Route: task2/generateToken]
in APPPATH\Controllers\Task2.php on line 46.
 1 SYSTEMPATH\Router\Router.php(562): CodeIgniter\Router\AutoRouter->getRoute('task2/generateToken', 'GET')
 2 SYSTEMPATH\Router\Router.php(227): CodeIgniter\Router\Router->autoRoute('task2/generateToken')
 3 SYSTEMPATH\CodeIgniter.php(834): CodeIgniter\Router\Router->handle('task2/generateToken')
 4 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 07:00:34 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
[Method: GET, Route: task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 59.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 07:01:47 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
[Method: GET, Route: task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 59.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 07:04:22 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
[Method: GET, Route: task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 59.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 07:05:59 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
[Method: GET, Route: task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 60.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 07:08:40 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
[Method: GET, Route: task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 60.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 07:08:48 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
[Method: GET, Route: task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 60.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 07:08:58 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
[Method: GET, Route: Task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 60.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 07:09:48 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
[Method: GET, Route: Task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 60.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 07:13:36 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
[Method: GET, Route: Task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 61.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 07:13:56 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
[Method: GET, Route: Task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 61.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 07:16:08 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
[Method: GET, Route: Task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 61.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 07:16:09 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
[Method: GET, Route: Task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 61.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 07:17:18 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
[Method: GET, Route: Task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 59.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-04-24 07:17:19 --> Error: Firebase\JWT\JWT::decode(): Argument #3 ($headers) cannot be passed by reference
[Method: GET, Route: Task2/getUserDetails]
in APPPATH\Controllers\Task2.php on line 59.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Task2->getUserDetails()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Task2))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
